src/components/dashboard/DeleteDealButton.tsx

```diff
@@ -25,7 +29,7 @@ export default function DeleteDealButton({ id }: DeleteDealButtonProps) {
   }
   return (
     <Button variant="outline-destructive" onClick={handleDeleteClick}>
-      Delete
+      <FaTrash className="inline h-3" />
```

Comment:
Probably not necessary here, but to be more accessible you can add button text and use the `sr-only` so it renders for screen readers.

Reply:
+1 for this recommendation.